tests/test_interface.rb

```diff
@@ -75,6 +75,14 @@ def interface_count
     arr.count
   end
 
+  # Helper to check for misc speed change disallowed error messages.
+  def speed_change_disallowed(message)
+    pattern = Regexp.new('(port doesn t support this speed|' \
```

Comment:
Just to make sure, the error message you're matching against is missing an apostrophe in `doesn't`?